index.html

```diff
@@ -144,10 +144,29 @@ <h2>Terminology</h2>
 <section id="object-spec">
     <h2>Object Specification</h2>
     <p>
-        An OCFL Object is a group of one or more content bitstreams (data and metadata), and their administrative
-        information that are together identified by a URI. The object may contain a sequence of versions of the
-        bitstreams that represent the evolution of the object's contents.
+        An OCFL Object is a group of one or more files, directories, and administrative
+        information, that are together identified by a URI. The object may contain a sequence of versions of the
+        files that represent the evolution of the object's contents.
     </p>
+    <p>
+        A file is defined as a content bitstream that can be stored and transmitted. Directories (also called "folders")
+        allow for the organization of files into tree-like hierarchies. The content of an OCFL Object is the files and
+        directories that are stored <i>within</i> the hierarchy layout provided as part of the spec.
```

Comment:
Same issue with directories as above

Reply:
but isn't this section defining what a directory is? i guess i'm not sure how you can define a directory without using the word directory, or maybe i'm misunderstanding the comment (probably this).

Reply:
I think this still has the problem that it implies directories are an element of content whereas we support them only in so much as they are where the files live. I propose following @rosy1280's suggested construction above...

> The content of an OCFL Object is the files and the directory hierarchy they are organized in.

though that is pretty much redundant with the definition above. Maybe just the first two sentences of @ahankinson's proposal are enough?